website/api/helpers/get-extended-osquery-schema.js

```diff
@@ -149,7 +149,7 @@ module.exports = {
                 for(let platform of columnHasFleetOverrides.platforms) {
                   if(platform === 'darwin') {
                     platformWithNormalizedNames.push('macOS');
-                  } else if(platform === 'chromeos') {
+                  } else if(platform === 'chrome') {
```

Comment:
Is this what it is in the REST API?  We should use the same identifier everywhere.  @lukeheath @rachaelshaw

Reply:
@mikermcneil The `platform` for ChromeOS hosts is `chrome` in the REST API.

This screenshot is from a response from the [host summary](https://fleetdm.com/docs/using-fleet/rest-api#get-hosts-summary) API endpoint:
<img width="280" alt="image" src="https://github.com/fleetdm/fleet/assets/7445991/0a6f2176-ffe1-49e5-9489-2e6b9b55090c">

Reply:
Confirmed `chrome` is standardized in the API.